build.gradle

```diff
@@ -104,6 +104,7 @@ dependencies {
     compile group: "org.jetbrains", name: "annotations", version: annotations_version
     compile group: "org.jetbrains.kotlinx", name: "kotlinx-coroutines-core", version: coroutines_version
     compile group: "org.jetbrains.kotlinx", name: "kotlinx-coroutines-jdk8", version: coroutines_version
+    compile 'com.google.cloud:google-cloud-translate:1.95.0'
```

Comment:
here

Reply:
you need to also include it in the shadowjar

Reply:
https://github.com/kami-blue/client/pull/1009#discussion_r449668651